spec/util/roomValidate-spec.js

```diff
@@ -0,0 +1,102 @@
+var rooms = {
```

Comment:
Maybe move username related validations to a separate spec than room validate.  Also a test for the blacklist of usernames would be good.  Those names should probably be in a file that we dynamically load like the config file.  I can do any of these mentioned things, just noting things.